src/tiledimage.js

```diff
@@ -65,6 +65,7 @@
  * @param {Boolean} [options.alwaysBlend] - See {@link OpenSeadragon.Options}.
  * @param {Number} [options.minPixelRatio] - See {@link OpenSeadragon.Options}.
  * @param {Number} [options.opacity=1] - Opacity the tiled image should be drawn at.
+ * @param {String} [options.compositeOperation='source-over'] - How a tiled source image are drawn onto an existing image.
```

Comment:
How about "How the image is composited onto other images; see compositeOperation in {@link OpenSeadragon.Options} for possible values."